CDA1.json

```diff
@@ -0,0 +1,1324 @@
+{
```

Comment:
Unknown value origins: 
![grafik](https://user-images.githubusercontent.com/25061876/223135923-77097ac4-a279-4b58-b213-2409d3ce1d65.png)

Reply:
@Christoph27 That's my only comment on this PR - the rest looks good!

Reply:
value origins adapted now